src/sumBy.ts

```diff
@@ -19,6 +26,10 @@ export function sumBy<T>(
   callbackfn: (value: T, index: number, data: ReadonlyArray<T>) => number,
 ): (items: ReadonlyArray<T>) => number;
 
+export function sumBy<T>(
+  callbackfn: (value: T, index: number, data: ReadonlyArray<T>) => bigint,
+): (items: ReadonlyArray<T>) => bigint | 0;
```

Comment:
**suggestion:** let's have a stricter type here; if the array is non-empty we know that the output type is `bigint`, not just `bigint | 0`

Reply:
thx for your suggestion. I have added the type tests